patches/extensions-common-api-_api_features.json.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/extensions/common/api/_api_features.json b/extensions/common/api/_api_features.json
+index c73061c8fec8f419226cfc825635596d39ab6484..92eee4d4b0d186c7133cd2b1db6f6f68d1477e36 100644
+--- a/extensions/common/api/_api_features.json
++++ b/extensions/common/api/_api_features.json
+@@ -526,6 +526,7 @@
+     "contexts": ["webui"],
+     "matches": [
+       "chrome://password-change/*"
++      ,"chrome://newtab/*"
```

Comment:
Enables `chrome.runtime.sendMessage` from the New Tab Page @brave/patch-reviewers

Reply:
If this is ok with the security reviewers, the patch LGTM.

Reply:
sgtm as long as the receiver validates `sender.id` in `chrome.runtime.onMessage` (to make sure it's coming from the expected extension)

Reply:
i see this allows any `chrome://` URL - https://github.com/brave/brave-core/pull/6192/files#diff-4263de6a462535357156da5fb6faf5e8c5adaf5404bde45194da89d5e93f19c4R33-R38. this should be ok as long as extensions can't register `chrome://` URLs. it'd be more restrictive to filter it to just ID if possible.

Reply:
Well, there is no `id` since the NTP isn't an extension, so if we want to further restrict then we can also check the hostname (`newtab` in this case). But extensions or apps absolutely should not be able to register or run any code on `chrome://`.

Reply:
sgtm then